src/graphic/Fast3D/gfx_wiiu.cpp

```diff
@@ -39,7 +39,7 @@
 
 #include <ImGui/backends/wiiu/imgui_impl_wiiu.h>
 #include "port/wiiu/WiiUImpl.h"
-#include "libultraship/libultraship.h"
+#include "libultraship/classes.h"
```

Comment:
needed this because of double def issues between `coreinit/time` and `coreinit/thread` and lus provided stuff